### PR TITLE
translation-DatePicker

### PR DESCRIPTION
**Pull request by ddcat1115**
First of all, thanks for your contribution! :-)

Please makes sure these boxes are checked before submitting your PR, thank you!
- [x] Make sure you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
- [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
- [x] Rebase before creating a PR to keep commit history clear.
- [x] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

**Comment by ddcat1115** [comment]
#1471

**Comment by ddcat1115** [comment]
@manV please review this translation, thanks! 😃

File: components/date-picker/demo/disabled-date.md [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
+
 设置 `disabledDate` 方法，来确定不可选时段。
 
 如上例：不可选择今天之后的日期。
 
+## en-US
+
+Specify unselectable period by `disabledDate`.
+
+As in the examples above: you can't select a date later than today.
```
Comment (manV) [inline components/date-picker/demo/disabled-date.md]: As in the **example** above: you can't select a date later than today.

File: components/date-picker/demo/locale.md [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
+  en-US: Locale
 ---
 
+## zh-CN
+
 通过 `locale` 配置时区、语言等, 默认支持 `en_US`，`zh_CN`。
 
+## en-US
+
+To set the properties like time zone, language and etc. en_US, zh_CN are supported by default.
```
Comment (manV) [inline components/date-picker/demo/locale.md]: Use `locale` to set the properties like time zone, language and etc. `en_US`, `zh_CN` are supported by default.

File: components/date-picker/index.en-US.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+|--------------|----------------|----------|--------------|
+| value        | to set date           | String/Date   | -           |
+| defaultValue | to set default date        | String/Date   | -           |
+| format       | to set the date format, refer to [GregorianCalendarFormat](https://github.com/yiminghe/gregorian-calendar-format) | String   | "yyyy-MM-dd" |
+| disabledDate | to specify the date that cannot be selected | function | -           |
+| onChange     | a callback function, can be executed when the selected time is changing | function(date, dateString) | -           |
+| disabled     | determine whether the DatePicker is disabled           | Boolean     | false        |
+| style        | to customize the style of the input box     | Object     | {}   |
+| popupStyle   | to customize the style of the popup calendar   | Object     | {}   |
+| size         | determine the size of the input box, the height of `large` and `small`, were 32px and 22px respectively, while default size is 28px | String   | -  |
```
Comment (manV) [inline components/date-picker/index.en-US.md]: | size         | determine the size of the input box, the height of `large` and `small`, **are** 32px and 22px respectively, while default size is 28px | String   | -  |

File: components/date-picker/index.en-US.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+|--------------|----------------|----------|--------------|
+| value        | to set date          | String/Date   | -           |
+| defaultValue | to set default date       | String/Date   | -           |
+| format       | to set the date format, refer to [GregorianCalendarFormat](https://github.com/yiminghe/gregorian-calendar-format) | String   | "yyyy-MM" |
+| disabledDate | to specify the date that cannot be selected | function | -           |
+| onChange     | a callback function, can be executed when the selected time is changing | function(Date value) | -           |
+| disabled     | determine whether the MonthPicker is disabled           | Boolean     | false        |
+| style        | to customize the style of the input box     | Object     | {}   |
+| popupStyle   | to customize the style of the popup calendar   | Object     | {}   |
+| size         | determine the size of the input box, the height of `large` and `small`, were 32px and 22px respectively, while default size is 28px | String   | -  |
```
Comment (manV) [inline components/date-picker/index.en-US.md]: | size         | determine the size of the input box, the height of `large` and `small`, **are** 32px and 22px respectively, while default size is 28px | String   | -  |

**Comment by ddcat1115** [comment]
@manV    Thanks! And sorry for the careless mistakes 😅

**Comment by manV** [comment]
@ddcat1115 that's what I am here for :smile:

**Comment by benjycui** [comment]
@ddcat1115 根据 review 更新下？

**Comment by ddcat1115** [comment]
fixed already

**Comment by benjycui** [comment]
Great! THX!

**Pull request merged**